news/2023/2023-08-28-medal-hunter-titles-second-coming.md

```diff
@@ -0,0 +1,70 @@
+"ith a lot of guesses and such an insane 2 hints, 1 being slightly misleading, half of medal hunters entered full Sherlock Holmes mode with the possibilities of the solution, relying more on insanity than logic and drowning the discussion threads with abominations of theories. Some were genuinely good theories that Ephemeral would take note of, and others... well, let's just say Pac-Man theory.
+
+![](/wiki/shared/news/2023-08-28-medal-hunter-titles-second-coming/pacman_theory.png)
+
+But, despite all the challenges the community was still determined to find the solution. One of them was [UberFazz](https://osu.ppy.sh/users/8646059), who was going back and forth, seeking for the way to get this medal. At first, they assumed it would be achieving 16,777,216 (256 to the 3rd power) score, because that's how many possible colors there are in RGB. Another guess of theirs was to interact with osu! website to find a hidden message somewhere on it, which would then lead to a solution. They were also told by one of developers they had insanely close guesses multiple times, but that was just not enough...
+
+The community was trying their best to *hunt* for the solution, but after an enormous number of attempts that they thought would surely work, insanity started creeping in, and hope was slowly starting to fade. 
+
+But suddenly, after more than 3 weeks, the very last hint was dropped by [Repflez](https://osu.ppy.sh/users/201392) saying that "a message in the last 8 hours was a hint in the right direction", and after reading it, UberFazz found that message and figured exactly what they needed to do. That theory was to set scores with particular combos that relate to the RGB values of a difficulty, which UberFazz guessed was on on the celinsane difficulty as it literally meant cell + insane just like the medal suggested. Shortly after trying this idea in regards to combo matching each value of the RGB, UberFazz successfully got **Internment** medal and received the "**Medal Hunter**" title, after more than a month since the medal's release!
+
+![](/wiki/shared/news/2023-08-28-medal-hunter-titles-second-coming/uberfazz_profile.png)
+
+And after giving it more tests which crossed out grades from the equation and being celinsane diff specific, the community was finally ready to present the full solution to **Internment** medal:
+
+On any "Insane" difficulty (4.0\* - 5.29\*) of Frums - theyaremanycolors, set three plays in a row that have combos equal to the R, G, and B values of the difficulty indicator (on the osu! website) for the map you are playing. The possible combinations are as follows:
```

Comment:
```suggestion
On any "Insane" difficulty (4.0\* - 5.29\*) of [Frums - theyaremanycolors](https://osu.ppy.sh/beatmapsets/1416051#osu/2918249), set three plays in a row that have combos equal to the R, G, and B values of the difficulty indicator (on the osu! website) for the map you are playing. The possible combinations are as follows:
```

Reply:
fair, but will do it manually because diff isn't correct